app/collins/util/config/GpuConfig.scala

```diff
@@ -0,0 +1,12 @@
+package collins.util.config
+
+object GpuConfig extends Configurable {
+  override val namespace = "gpu"
+  override val referenceConfigFilename = "gpu_reference.conf"
+
+  def gpuVendors= getStringSet("gpuVendors")
```

Comment:
I would like the default values to be reflected here, instead of relying on the config to include something. I.e. the default unpopulated config should include nvidia, instead of the user being forced to set this config on every installation